<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>cls</class>
 <widget class="QWidget" name="cls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>726</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Image Classification</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    font-family: 'Segoe UI', sans-serif;
    font-size: 10pt;
}
QGroupBox {
    border: 1px solid #d0d0d0;
    border-radius: 5px;
    margin-top: 1ex;
    padding-top: 15px;
}
QGroupBox::title {
    subcontrol-origin: margin;
    subcontrol-position: top center;
    padding: 0 5px;
    background: transparent;
}
QSplitter::handle {
    background: #c0c0c0;
    border: 1px solid #a0a0a0;
    border-radius: 2px;
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <widget class="QGroupBox" name="autoMode">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="3" column="0">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QCheckBox" name="automode">
          <property name="text">
           <string>Auto Mode</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Minimum</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="2">
         <widget class="QLineEdit" name="confidenceEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>0.5</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="conf">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>置信度</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>模型选择</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QComboBox" name="comboBox">
          <property name="font">
           <font>
            <family>Segoe UI,sans-serif</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <item>
           <property name="text">
            <string>cls</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>detection</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QSplitter" name="mainSplitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="handleWidth">
      <number>8</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="contentContainer">
      <layout class="QHBoxLayout" name="contentLayout">
       <property name="spacing">
        <number>15</number>
       </property>
       <item>
        <widget class="QGroupBox" name="labelGroup">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>150</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="labelLayout">
          <item>
           <widget class="QCheckBox" name="one">
            <property name="text">
             <string>1</string>
            </property>
            <property name="shortcut">
             <string>1</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="two">
            <property name="text">
             <string>2</string>
            </property>
            <property name="shortcut">
             <string>2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="three">
            <property name="text">
             <string>3</string>
            </property>
            <property name="shortcut">
             <string>3</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="four">
            <property name="text">
             <string>4</string>
            </property>
            <property name="shortcut">
             <string>4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="five">
            <property name="text">
             <string>5</string>
            </property>
            <property name="shortcut">
             <string>5</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="outpost">
            <property name="text">
             <string>outpost</string>
            </property>
            <property name="shortcut">
             <string>6</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="guard">
            <property name="text">
             <string>guard</string>
            </property>
            <property name="shortcut">
             <string>7</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="base">
            <property name="text">
             <string>base</string>
            </property>
            <property name="shortcut">
             <string>8</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">classButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="DimensionEdit">
         <property name="title">
          <string/>
         </property>
         <layout class="QVBoxLayout" name="extensionLayout">
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QWidget" name="widget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>250</height>
             </size>
            </property>
            <property name="cursor">
             <cursorShape>PointingHandCursor</cursorShape>
            </property>
            <widget class="QLabel" name="label">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>80</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Width</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="widthEdit">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>80</y>
               <width>113</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QLabel" name="label_2">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>120</y>
               <width>71</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Height</string>
             </property>
            </widget>
            <widget class="QLineEdit" name="heightEdit">
             <property name="geometry">
              <rect>
               <x>100</x>
               <y>120</y>
               <width>113</width>
               <height>25</height>
              </rect>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
            <widget class="QPushButton" name="sure">
             <property name="geometry">
              <rect>
               <x>30</x>
               <y>180</y>
               <width>101</width>
               <height>31</height>
              </rect>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
    border: 1px solid #c0c0c0;
    border-radius: 5px;
    background: #f0f0f0;
    padding: 5px;
}
QPushButton:hover {
    background: #e0e0e0;
}
QPushButton:pressed {
    background: #d0d0d0;
}</string>
             </property>
             <property name="text">
              <string>确定</string>
             </property>
            </widget>
            <widget class="QRadioButton" name="autoCut">
             <property name="geometry">
              <rect>
               <x>10</x>
               <y>30</y>
               <width>361</width>
               <height>31</height>
              </rect>
             </property>
             <property name="text">
              <string>启用自动裁剪(会根据你自定义的宽高自动改变图像大小)</string>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="imageArea">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="previewLayout">
       <item>
        <widget class="QLabel" name="previewLabel">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>200</height>
          </size>
         </property>
         <property name="text">
          <string>Preview</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="saveLabel">
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton {
    border: 1px solid #c0c0c0;
    border-radius: 5px;
    background: #f0f0f0;
    padding: 5px;
}
QPushButton:hover {
    background: #e0e0e0;
}
QPushButton:pressed {
    background: #d0d0d0;
}</string>
         </property>
         <property name="text">
          <string>保存标签</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="classButtonGroup"/>
 </buttongroups>
</ui>
